File: EmbyStat.Services/Models/Show/ShowCollectionRow.cs
using System;
using System.Collections.Generic;
using EmbyStat.Common.Models.Show;

namespace EmbyStat.Services.Models.Show
{
    public class ShowCollectionRow
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Tvdb { get; set; }
        public string Imdb { get; set; }
        public double Size { get; set; }
        public string Banner { get; set; }
        public int Seasons { get; set; }
        public int Episodes { get; set; }
        public int Specials { get; set; }
        public int MissingEpisodeCount { get; set; }
        public IEnumerable<VirtualSeason> MissingEpisodes { get; set; }
        public DateTimeOffset? PremiereDate { get; set; }
        public bool Status { get; set; }
        public string SortName { get; set; }
    }
}

File: EmbyStat.Web/ClientApp/src/app/shared/models/show/show-collection-row.ts
import { Season } from './season';

export class ShowCollectionRow {
    id: string;
    title: string;
    seasons: number;
    episodes: number;
    missingEpisodeCount: number;
    missingEpisodes: Season[];
    premiereDate: Date;
    status: boolean;
    sortName: string;
    specials: number;
    percentage: number;
    tvdb: string;
    imdb: string;
    size: number;
    banner: string;
}
